uber/site_sections/emails.py

```diff
@@ -71,3 +71,25 @@ def test_email(self, session, subject=None, body=None, from_address=None, to_add
     def approve(self, session, subject):
         session.add(ApprovedEmail(subject=subject))
         raise HTTPRedirect('pending?message={}', 'Email approved and will be sent out shortly')
+
+    def emails_by_interest(self):
+        return {}
+
+    @csv_file
+    def emails_by_interest_csv(self, out, session, *args, **kargs):
```

Comment:
Not a big deal, but ``kwargs`` is a more conventional name for this than ``kargs``.  Like, ``kwargs`` is what you'll see in every Python tutorial.  Though we use ``**params`` throughout most of the Uber/RAMS codebase, to indicate that they're specifically HTTP query parameters.

We also don't need ``args`` here, since CherryPy passes HTTP parameters soleby by keyword argument, so this should probably just say ``, **params``.